dash/testing/browser.py

```diff
@@ -110,29 +110,32 @@ def wait_for_element(self, css_selector, timeout=None):
 
     # keep these two wait_for API for easy migration
     def wait_for_element_by_css_selector(self, selector, timeout=None):
+        _time = timeout or self._wait_timeout
         return self._wait_for(
             EC.presence_of_element_located,
             ((By.CSS_SELECTOR, selector),),
             timeout,
-            "timeout {} => waiting for selector {}".format(timeout, selector),
+            "timeout {}s => waiting for selector {}".format(_time, selector),
```

Comment:
nice fix on the timeout
🐄  minor, I would use more pythonic way format `(timeout if timeout else self._wait_timeout, selector)`

Reply:
🐄 in https://github.com/plotly/dash/pull/761/commits/55274bea6ad084443a785ca6c1bef926e9de0060